### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by AlineTalhouk** [TalhoukLab/OV_Histotypes#65]
I don't agree with this statement below figure 4.8 can you please explain:

Misclassified cases from a previous step of the sequence of classifiers are not included in subsequent steps of the training set CV folds. Thus, we cannot piece together the test set predictions from the sequential and two-step algorithms to obtain overall metrics.

Also I am assuming that figure 4.8 is on confirmation data? Please clarify that in a title somewhere.
